### ISSUE TITLE
Adjust battery colors

### ISSUE THREAD

**Issue by philcali** [philcali/pebble-carwings#2]
Right now we split it up in fourths
- greater than or equal 75% = green
- greater than or equal 50% = yellow
- greater than or equal 25% = orange
- less than 25%  = red

Replace it with three colors in thirds:
- greater than 8 = green
- greater than 4 = yellow
- less than or equal to 4 = red

Along with this change we need new colors to denote the type of charge the vehicle is receiving:
- cyan for quick
- blue for normal
- pink for trickle

Blocked by [PW-4](https://github.com/philcali/proxy-wings/issues/4).

**Comment by philcali** [comment]
Closed with #4
